homeassistant/components/mystrom/light.py

```diff
@@ -121,6 +141,16 @@ def is_on(self):
         """Return true if light is on."""
         return self._state
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info for the light entity."""
+        return DeviceInfo(
+            identifiers={(DOMAIN, format_mac(self._mac))},
```

Comment:
Note: I think it's OK to format it in the device info, but I think it shouldn't be formatted for the unique_id unless you implement a migration.

Reply:
changed to mac